plugins/BEdita/Core/tests/TestCase/Filesystem/Adapter/LocalAdapterTest.php

```diff
@@ -23,6 +23,24 @@
  */
 class LocalAdapterTest extends TestCase
 {
+    /**
+     * Temporary test files path, removed after test
+     * @var string
+     */
+    const FILES_PATH = WWW_ROOT . 'static-files' . DS . 'subdir';
```

Comment:
In `tests/bootstrap.php` we configure `LocalAdapter` to use `plugins/BEdita/Core/tests/uploads` as its root. That configuration is not used in this case, but we might use the same path…